packages/slo-stat-bot/test/slo-stat-label.ts

```diff
@@ -0,0 +1,345 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import {resolve} from 'path';
+import {Probot} from 'probot';
+import nock from 'nock';
+import * as fs from 'fs';
+import * as assert from 'assert';
+import {describe, it, beforeEach, afterEach} from 'mocha';
+
+// eslint-disable-next-line node/no-extraneous-import
+import Webhooks from '@octokit/webhooks';
+import snapshot from 'snap-shot-it';
+import handler from '../src/slo-stat-label';
+import spies from 'chai-spies';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const chai = require('chai');
+chai.use(spies);
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const sinon = require('sinon');
```

Comment:
Is there a reason you did this as opposed to `import sinon from 'sinon''`?

Reply:
I am unable to import it because I cannot create a sandbox for easily restoring any stubs or fakes. When using import sinon from 'sinon' it is expecting us to create a SinonSpy<any,any>. I can change my code to define stubs individually w/o using sandbox if needed.

Reply:
Looking through all the `sandbox` docs, it looks like `sandbox `is probably too heavyweight for what we need to use it for. I'd replace the `sandbox` with a single stub. I'll comment more down below.